packages/sw-routing/src/lib/error-factory.js

```diff
@@ -16,9 +16,9 @@
 import ErrorFactory from '../../../../lib/error-factory';
 
 const errors = {
-  'express-route-requires-absolute-path': `When using ExpressRoute, you must
-    provide a path that starts with a '/' character. You can only match
-    same-origin requests. For more flexibility, use RegExpRoute.`,
+  'express-route-invalid-path': `When using ExpressRoute, you must
+    provide a path that starts with a '/' character (to match same-origin
+    requests) or that starts with 'https://' (to match cross-origin requests)`,
```

Comment:
It can also start with 'http://' and what is this error thrown? If it's when the route can't be passed, it may be worth just saying the route must be a valid URL starting with a '/' or 'http://' and leave out the same-origin / cross-origin request details (I'm just thinking that this message is overly verbose).